akka-javasdk/src/main/java/akka/javasdk/Metadata.java

```diff
@@ -213,13 +213,6 @@ public interface Metadata extends Iterable<Metadata.MetadataEntry> {
    */
   CloudEvent asCloudEvent(String id, URI source, String type);
 
-
-  /**
-   * Get the trace context associated with this request metadata.
-   * @return The trace context.
-   */
-  TraceContext traceContext();
```

Comment:
I think this was mention in the stand-up at some point, but I just want to recall. 

This is a breaking API change. What is the racional for not going through deprecation cycle?

Reply:
I expect nobody is using this deep-end-functionality yet, so saving a bit of work by not deprecation-cycling.